pkg/framework/aries/api/vdr/vdr.go

```diff
@@ -35,6 +41,9 @@ type VDR interface {
 	Read(did string, opts ...resolve.Option) (*did.DocResolution, error)
 	Store(doc *did.Doc, by *[]doc.ModifiedBy) error
```

Comment:
what's store vs update?

Reply:
@troyronda I will add TODO to remove Store